src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
-package core.basesyntax;
-
-public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
-
-    }
-}
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class WorkWithFile {
+
+    private List<String> readFromFile(String fromFileName) {
+        List<String> data;
+        File file = new File(fromFileName);
+        try {
+            data = Files.readAllLines(file.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read file", e);
+        }
+        return data;
+    }
+
+    private String balanceCalculate(List<String> data) {
+        int supplySum = 0;
+        int buySum = 0;
+        final String separator = System.lineSeparator();
+        for (String element : data) {
+            String[] elements = element.split(",");
+            if (elements[0].equals("supply")) {
+                supplySum += Integer.parseInt(elements[1]);
+            } else if (elements[0].equals("buy")) {
+                buySum += Integer.parseInt(elements[1]);
+            }
+        }
+        int result = supplySum - buySum;
+        return ("supply," + supplySum + separator
+                + "buy," + buySum + separator
+                + "result," + result);

```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)